packages/naming.cell.match/cell-match.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const assert = require('assert');
+
+const BemCell = require('@bem/sdk.cell');
+const bemNamingParse = require('@bem/sdk.naming.entity.parse');
+const pathPatternParser = require('@bem/sdk.naming.cell.pattern-parser');
+
+const ALPHANUM_RE = '[a-z][\\w\\-]*';
+const resc = s => String(s).replace(/[\\^$*+?.()|[\]{}]/g, '\\$&');
+
+const buildPathParseMethod = ({ fs: { pattern, delims: dd = {}, scheme = 'nested' }, delims, wordPattern: wp = ALPHANUM_RE }) => {
```

Comment:
Seems like we need a default preset. Now some defaults are located inside "naming.cell.match" package. It is not obvious